### ISSUE TITLE
Output of 'bosh deployments' should be able to come in json format

### ISSUE THREAD

**Issue by kevinzurek** [cloudfoundry/bosh#1665]
I am currently using the Bosh API to call /deployments to get a json output of 'bosh deployments'. Using the CLI you can only get an output that is very hard to parse. 

Or perhaps i've missed something and this can already be done 😄

**Comment by dpb587-pivotal** [comment]
Have you tried using the [new CLI](http://bosh.io/docs/cli-v2.html)? New version supports `--json` for some JSON output, but it is also more script-friendly with its `cat`/`awk`-friendly output. You might also find `--column=name`-type args useful, too.

**Comment by kevinzurek** [comment]
Much prettier, thank you @dpb587-pivotal . I was still using the old CLI. 👍
